OTAnalytics/application/datastore.py

```diff
@@ -4,6 +4,7 @@
 from typing import Iterable, Optional, Tuple
 
 from OTAnalytics.domain.event import Event, EventRepository
+from OTAnalytics.domain.flow import Flow, FlowId, FlowRepository
```

Comment:
Why do we not need observer classes for flows as we do for sections?

Reply:
To update the ui on added, removed or renamed flows.